runtime/executor/src/test/java/org/apache/nemo/runtime/executor/datatransfer/DataTransferTest.java

```diff
@@ -458,6 +463,9 @@ private void writeAndReadWithDuplicateData(final BlockManagerWorker sender,
       }
       dataReadList2.add(dataRead2);
     });
+    IntStream.range(0, PARALLELISM_TEN).forEach(dstTaskIndex -> {
+
+    });
```

Comment:
Were these lines added by mistake?

Reply:
I think I forgot to delete them. Thanks!